tests/test_job.py

```diff
@@ -17,3 +149,43 @@ def test_job_instantiation_with_extra_field(self, job):
         assert (
             new_job.new_field == "any_value"
         )  # The new value should be stored regardless
+
+    @pytest.mark.parametrize(
+        "filters",
+        [
+            {
+                "job_ids": [UUID(int=0x1), UUID(int=0x2)],
+                "project_id": [UUID(int=0x1), UUID(int=0x2)],
+                "user_id": ["1", "2"],
+                "batch_id": [UUID(int=0x1), UUID(int=0x2)],
+                "status": [JobStatus.DONE, JobStatus.PENDING],
+                "min_runs": 10,
+                "max_runs": 20,
+                "errors": True,
+                "origin": [JobOrigin.AZURE, JobOrigin.CLIENT],
+                "start_date": datetime(2023, 1, 1),
+                "end_date": datetime(2023, 1, 1),
+            }
```

Comment:
Here we have only one test case for all filters combined. Could they also be tested individually?